server/src/main/java/com/ibm/JavaProfilingLanguageServer/HotMethod.java

```diff
@@ -89,10 +87,10 @@ public Range findInTextDocument(TextDocumentItem textDocument) {
 
 			contextRange.setEnd(new Position(contextRange.getStart().getLine(), startOfMethodName + endOfMethodName));
 
-			//		System.out.println("*** FINAL SCOPE ***");
-			//		System.out.printf("Range: start: %s:%s, end: %s:%s%n",
-			//				contextRange.getStart().getLine(), contextRange.getStart().getCharacter(),
-			//				contextRange.getEnd().getLine(), contextRange.getEnd().getCharacter());
+			// System.out.println("*** FINAL SCOPE ***");
+			// System.out.printf("Range: start: %s:%s, end: %s:%s%n",
+			// 	contextRange.getStart().getLine(), contextRange.getStart().getCharacter(),
+			// 	contextRange.getEnd().getLine(), contextRange.getEnd().getCharacter());
```

Comment:
I know these were there before but shouldn't we be removing commented print lines?

Reply:
Removed